src/connector.js

```diff
@@ -0,0 +1,612 @@
+			`<section class="mrgn-tp-lg alert alert-warning">
+				<h2 tabindex="-1" id="wb-land" class="h4">Aucun résultat</h2>
+				<p>Aucun résultat ne correspond à vos critères de recherche.</p>
+				<p>Suggestions:</p>
+				<ul>
+					<li>Assurez-vous que tous vos termes de recherches sont bien orthographiés </li>
+					<li>Utilisez de différents termes de recherche </li>
+					<li>Utilisez des termes de recherche plus généraux </li>
+					<li>Consultez les&nbsp;<a href="/fr/sr/tr.html"> trucs de recherche </a></li>
+					<li>Essayez la <a href="/fr/chaires-recherche/rechercher/recherche-avancee.html">recherche avancée</a></li>
+				</ul>
+			</section>`;
+	}
+	else {
+		noResultTemplateHTML = 
+			`<section class="mrgn-tp-lg alert alert-warning">
+				 <h2 tabindex="-1" id="wb-land" class="h4">No results</h2>
+				 <p>No pages were found that match your search terms.</p>
+				 <p>Suggestions:</p>
+
+				 <ul>
+					<li>Make sure all search terms are spelled correctly</li>
+					<li>Try different search terms</li>
+					<li>Try more general search terms</li>
+					<li>Consult the&nbsp;<a href="/en/sr/st.html">search tips</a></li>
+					<li>Try the&nbsp;<a href="/en/sr/srb/sra.html">advanced search</a></li>
+				 </ul>
+			</section>`;
+	}
+}
+
+if ( !resultErrorTemplateHTML ) {
+	if ( lang == "fr" ) {
+		resultErrorTemplateHTML = 
+			`<section class="mrgn-tp-lg alert alert-warning">
+				 <h2 tabindex="-1" id="wb-land" class="h4">The Canada.ca Search is currently experiencing issues</h2>
+				 <p>A resolution for the restoration is presently being worked.	We apologize for any inconvenience.</p>
+			</section>`;
+	}
+	else {
+		resultErrorTemplateHTML = 
+			`<section class="mrgn-tp-lg alert alert-warning">
+				 <h2 tabindex="-1" id="wb-land" class="h4">The Canada.ca Search is currently experiencing issues</h2>
+				 <p>A resolution for the restoration is presently being worked.	We apologize for any inconvenience.</p>
+			</section>`;
+	}
+}
+
+if ( !querySummaryTemplateHTML ) {
+	if ( lang == "fr" ) {
+		querySummaryTemplateHTML = 
+			`<section>
+				 <h2 tabindex="-1" id="wb-land" class="h4">%[numberOfResults] résultats de recherche pour "%[query]"</h2>
+			</section>`;
+	}
+	else {
+		querySummaryTemplateHTML = 
+			`<section>
+				 <h2 tabindex="-1" id="wb-land" class="h4">%[numberOfResults] search results for "%[query]"</h2>
+			</section>`;
+	}
+}
+if ( !noQuerySummaryTemplateHTML ) {
+	if ( lang == "fr" ) {
+		noQuerySummaryTemplateHTML = 
+			`<section>
+				 <h2 tabindex="-1" id="wb-land" class="h4">%[numberOfResults] résultats de recherche</h2>
+			</section>`;
+	}
+	else {
+		noQuerySummaryTemplateHTML = 
+			`<section>
+				 <h2 tabindex="-1" id="wb-land" class="h4">%[numberOfResults] search results</h2>
+			</section>`;
+	}
+}
+
+if ( !previousPageTemplateHTML ) {
+	if (lang == "fr") {
+		previousPageTemplateHTML = 
+			`<a href="" rel="prev">Page précédente<span class="wb-inv">: Page précédente des résultats de recherche</span></a>`;
```

Comment:
By reading how it is built, it seem those pagination are acting like a button, please code them by using the button element instead of the anchor element.

It is also strange to have an empty href value for those anchor.

```suggestion
			`<button type="button">Page précédente<span class="wb-inv">: Page précédente des résultats de recherche</span></button>`;
```

some adjustment in the reference implementation (Add more working example) might by required

Reply:
> some adjustment in the reference implementation (Add more working example) might by required

Agreed and this will be treated when time allows, but not forgotten.

Reply:
MANDATORY